features/configuration_files/exclude_paths_directives.feature

```diff
@@ -22,6 +22,60 @@ Feature: Exclude paths directives
     When I run reek -c config.reek .
     Then it succeeds
     And it reports nothing
+
+  Scenario: Exclude directory with a trailing slash
+    Given a file named "bad_files_live_here/smelly.rb" with:
+      """
+      # A smelly example class
+      class Smelly
+        def alfa(bravo); end
+      end
+      """
+    When I run reek .
+    Then the exit status indicates smells
+    Given a file named "config.reek" with:
+      """
+      ---
+      exclude_paths:
+        - bad_files_live_here/
```

Comment:
Can we combine all these cases into a single scenario? I think it becomes a bit tedious otherwise.

Reply:
Done ;)